libs/s25main/SerializedGameData.cpp

```diff
@@ -101,8 +101,9 @@
 /// 8: noFlag::Wares converted to static_vector
 /// 9: Drop serialization of node BQ
 /// 10: troop_limits state introduced to military buildings
-/// 11:: wineaddon added, three new building types and two new goods
-static const unsigned currentGameDataVersion = 11;
+/// 11: wineaddon added, three new building types and two new goods
+/// 12: Add portrait index serialization in player state
+static const unsigned currentGameDataVersion = 12;
```

Comment:
You don't use this anywhere, do you? The player data is outside the game data and we don't have any versioning for that, do we? So I guess we can reset this and remove the checks if we go forward with this change

Reply:
Nah, I don't use it, it seems it was a blind change on my end years ago and I've just mindlessly upped the number in recent merge to make it relevant again.

I see that `Savegame::GetVersion()` returns a magic number - perhaps this should be upped and some kind of migration code added so that old stuff can be read properly?